src/commands/identity/enableSystemIdentity.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizard, type ExecuteActivityContext, type IActionContext } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../localize";
+import { type SystemIdentityTreeItemBase } from "../../tree/remoteProject/SystemIdentityTreeItemBase";
+import { type SlotTreeItem } from "../../tree/SlotTreeItem";
+import { createActivityContext } from "../../utils/activityUtils";
+import { EnableSystemIdentityAssignStep } from "./EnableSystemIdentityStep";
+import { type ManagedIdentityAssignContext } from "./ManagedIdentityAssignContext";
+
+export async function enableSystemIdentity(context: IActionContext, node: SystemIdentityTreeItemBase): Promise<undefined> {
+    const grandparentNode = node.parent.parent as SlotTreeItem;
+    const title: string = localize('enabling', 'Enabling system assigned identity for "{0}"...', grandparentNode.site.fullName);
```

Comment:
Should this say "Enable" instead of "Enabling" as well as no ellipses at the end?
![image](https://github.com/user-attachments/assets/4ea0fc7e-3eba-4696-a7cd-b636b5b92f5d)